### ISSUE TITLE
Missing gnome-tracker

### ISSUE THREAD

**Issue by RobertBuhren** [clearlinux/distribution#739]
Filtering files in firefox "Open Files" dialogue complains due to missing gnome-tracker.

Steps to reproduce:
Press "ctrl-o" in firefox and try to filter the result by typing.

![Screenshot from 2019-05-15 23-04-27](https://user-images.githubusercontent.com/815756/57809604-0eca0180-7766-11e9-9ab8-5d4e396b43cd.png)

**Comment by ahkok** [comment]
Disabled due to the negative effects of `gnome-tracker` on purpose. Not sure how we can solve this bug, therefore.

**Comment by ahkok** [comment]
Maybe allow users to manually enable tracker, perhaps? @mrkz ?

**Comment by miguelinux** [comment]
What Desktop Environment are you using?

Is it Gnome?

**Comment by RobertBuhren** [comment]
Yes, this is gnome. It seems that this issue is present with every gtk open dialogue.
